index.html

```diff
@@ -0,0 +1,686 @@
+                            <li><a class="rubric-bottom-list" href="#">Lorem ipsum dolor sit amet, ne adhuc suavitate.</a></li>
+                        </ul>
+                    </div>
+
+                </div>
+                <div class="rubric-thumbnail">
+                        <h2 class= "section-title"><a  class="section-title-a" href="#">Apps ></a></h2>
+
+                        <div class="rubric-background-container">
+                            <div class="second-tile-gradient">
+                            </div>
+                            <div class="rubric-thumbnail-head">
+                                <h4 class="most-popular">MOST POPULAR</h4>
+                                <h2><a class="rubric-first-title" href="#">Logitech built a 3-in-1 charger for Apple fans</a></h2>
+                                <h5 class="rubric-second-title"><a class="rubric-second-title" href="#">Napier Lopes</a> - 20 hours ago</h5>
+                            </div>
+                        </div>
+
+                        <div class="rubric-thumbnail-middle">
+                            <div class="rubric-middle-titles-container">
+                                <h4 class="rubric-middle-title1">LATEST</h4>
+                                <h4><a class="rubric-middle-title2" href="#">Google Play now has dark mode — here's how to turn it on</a></h4>
+                            </div>
+                            <img class="rubric-img" src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/03/google-play-store-dark-mode-796x417.jpg" alt="logo">
+                        </div>
+
+                        <div class="rubric-thumbnail-bottom">
+                            <ul class="rubric-ul">
+                                <li><a class="rubric-bottom-list" href="#">Lorem ipsum dolor sit amet, ne adhuc suavitate.</a></li>
+                                <li><a class="rubric-bottom-list" href="#">Lorem ipsum dolor sit amet, ne adhuc suavitate.</a></li>
+                                <li><a class="rubric-bottom-list" href="#">Lorem ipsum dolor sit amet, ne adhuc suavitate.</a></li>
+                                <li><a class="rubric-bottom-list" href="#">Lorem ipsum dolor sit amet, ne adhuc suavitate.</a></li>
+                                <li><a class="rubric-bottom-list" href="#">Lorem ipsum dolor sit amet, ne adhuc suavitate.</a></li>
+                            </ul>
+                        </div>
+                </div>
+                <div class="rubric-thumbnail">
+                </div>
+                
+        </div>
+
+        <div class="latest-deals general-container">
+
+                    <div class="latest-deals-title">
+                        <h2 class="section-title"><a class="section-title-a" href="#">Latest Deals ></a></h2>
+                        <h3><a class="funding-navbar-a last-nav-item" href="#">See all Deals</a></h3>
+                    </div>
+
```

Comment:
- [ ] This section has larger images in the mobile version too. Once you fix the Latest news, you will fix this one if they have the same classes.